src/index.ts

```diff
@@ -382,6 +402,16 @@ export default class Matic extends SDKClient {
     return this.withdrawManager.processExits(tokenAddress, options)
   }
 
+  fastExit(exitNFT: string, signature: string, fastExitAddress: string, options?: SendOptions) {
+    if (!exitNFT || !signature || !fastExitAddress) {
+      throw new Error(`txHash,signature or fastExitAddress not provided`)
+    }
+    if (options && !options.from) {
+      throw new Error(`options.from is missing`)
+    }
```

Comment:
This condition is false if user didnt pass options object. You are using `options.from` in `safeTransferFrom` so it is always needed.